.github/actions/upstream-test/action.yml

```diff
@@ -59,7 +59,7 @@ runs:
             }}-double_down_${{
               inputs.double_down_version
             }}
-          stages: base, external_deps, hdf5, moab, dagmc
+          stages: base, external_deps, dagmc
           server-stage: dagmc_test
```

Comment:
How it works? I have noticed it just caching everything. And tests are already passed on the final stage. Maybe I have misconfigured something?

Reply:
What do you expect to be different?